### PR TITLE
Garbage collecting

### PR DESCRIPTION
**Pull request by abheek9**
- added missing function in app chron for recalculating segments with message components
- this should fix issue with phantom mongo collections

### CODE REVIEW

File: packages/server/src/api/segments/segments.service.ts [hunk 1]
```diff
@@ -413,6 +414,12 @@ export class SegmentsService {
         //console.log(`Deleted collection: ${collectionName}`);
       }
     } catch (error) {
```
Comment (mcharawi) [inline packages/server/src/api/segments/segments.service.ts:416]: Log this error here.

File: packages/server/src/api/segments/segments.service.ts [hunk 2]
```diff
@@ -751,6 +770,12 @@ export class SegmentsService {
         //await this.connection.db.collection(customersInSegment).drop();
         //console.log('Collection dropped successfully');
       } catch (e) {
```
Comment (mcharawi) [inline packages/server/src/api/segments/segments.service.ts:777]: Log this error here.

**Review by mcharawi — CHANGES_REQUESTED** [review]

File: packages/server/src/api/segments/segments.service.ts (hunk 1 above)
Comment (abheek9) [reply packages/server/src/api/segments/segments.service.ts:416]: added

File: packages/server/src/api/segments/segments.service.ts (hunk 2 above)
Comment (abheek9) [reply packages/server/src/api/segments/segments.service.ts:777]: added

**Pull request merged**